torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -625,6 +630,22 @@ def __init__(
         device_id: Optional[Union[int, torch.device]] = None,
         sync_module_states: bool = False,
     ):
+        if isinstance(auto_wrap_policy, ParamExecOrderWrapPolicy):
```

Comment:
Add a comment here mentioning that initial FSDP wrapping / initialization is done with auto_wrap_policy.initial_policy?